### PR TITLE
UTOPIA-782: Tekton Objects

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

Adding templates for tekton objects used in emerald

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [ ] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: openshift/templates/tekton-pipeline/api/pipeline.yaml [hunk 1]
```diff
@@ -1,44 +1,36 @@
...
-    value: low
-  
+        name: process-apply-dpia-service
+      params:
+        - name: environment
+          value: $(params.environment)
+        - name: application
+          value: $(params.application)
+        - name: image
+          value: $(params.image)
```
Comment (akroon3r) [inline openshift/templates/tekton-pipeline/api/pipeline.yaml:36]:
```suggestion
          value: $(params.image)

```

File: openshift/templates/tekton-pipeline/api/service.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - 172.30.245.85
+  type: ClusterIP
+  ipFamilyPolicy: SingleStack
+  sessionAffinity: None
+  selector:
+    app.kubernetes.io/managed-by: EventListener
+    app.kubernetes.io/part-of: Triggers
+    eventlistener: dpia-deploy
+status:
+  loadBalancer: {}
```
Comment (akroon3r) [inline openshift/templates/tekton-pipeline/api/service.yaml:41]:
```suggestion
  loadBalancer: {}

```

Comment (akroon3r) [inline openshift/templates/tekton-pipeline/api/service.yaml:41]:
```suggestion
  loadBalancer: {}

```

File: openshift/templates/tekton-pipeline/api/trigger-template.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+apiVersion: triggers.tekton.dev/v1beta1
+kind: TriggerTemplate
+metadata:
+  name: process-apply-dpia-service
+spec:
+  params:
+  - name: environment
+    description: What environment is being targetted?
+  - name: application
+    description: blah
```
Comment (Bottle7) [inline openshift/templates/tekton-pipeline/api/trigger-template.yaml:10]: blah?

**Review by Bottle7 — APPROVED** [review]

**Pull request merged**